### PR TITLE
Update class-components.md

### PR DESCRIPTION
**Pull request by tigt**
## Checklist:

- [ ] I have read the **CONTRIBUTING** document and have signed (or will sign) the CLA.
- [ ] I have updated/added documentation affected by my changes.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: docs/class-components.md [hunk 1]
```diff
@@ -149,26 +155,27 @@ module.exports = {
 
 ## Split components
 
-Split components allow you to optimize for the case where a component is rendered on the server, but doesn't need to be re-rendered in the browser. Because the component doesn't need to be rendered in the browser, the template does not need to be sent to the browser. This can reduce your page weight by a few hundred bytes in some cases.
+Split components optimize for when a component renders on the server, and doesn’t need to dynamically rerender in the browser. As a result, its template and logic aren’t sent to the browser, reducing load time and download size.
```
Comment (mlrawlings) [inline docs/class-components.md]: :+1:

File: docs/class-components.md [hunk 2]
```diff
@@ -188,11 +195,11 @@ class {
 }
 ```
 
-Any javascript code related to the DOM or browser should also be inside the `component-browser.js` file.
+Any JavaScript code related to the DOM or browser should also be inside `component-browser.js`.
 
 ### Example
 
-_index.marko_
+`index.marko`
```
[the remark this answers is not in the document]
Comment (tigt) [reply docs/class-components.md]: Oh, cool. I’ll restore it.

File: docs/class-components.md [hunk 3]
```diff
@@ -403,131 +420,128 @@ The `:scoped` modifier on an attribute allows you to reference another element a
...
-An array of the root [HTML elements](https://developer.mozilla.org/en-US/docs/Web/API/element) that the component is bound to.
+An array of the root [`HTMLElement` objects](https://developer.mozilla.org/en-US/docs/Web/API/element) that the component is bound to.
 
-> `this.el` and `this.els` are deprecated. Please `this.getEl()` or `this.getEls()` methods. see [Methods](#getelkey-index)
+> ⚠️ `this.el` and `this.els` are deprecated. Please use [the `this.getEl()` or `this.getEls()` methods](#getelkey-index).
 
 ### `this.id`
 
-The String ID of the root [HTML element](https://developer.mozilla.org/en-US/docs/Web/API/element) that the component is bound to.
+A string identifier for the root HTML element that the component is bound to. (Not the `id` attribute.)
```
[the remark this answers is not in the document]
Comment (tigt) [reply docs/class-components.md]: Like the one used in `:scoped` by default?

[the remark this answers is not in the document]
Comment (DylanPiercey) [reply docs/class-components.md]: Yeup that is right.

**Comment by mlrawlings** [comment]
I'm going to go ahead merge this because it's a big improvement over what's there.  We can deal with these minor items later.

**Pull request merged**